Code/Framework/AzFramework/AzFramework/PaintBrush/PaintBrushSettings.cpp

```diff
@@ -19,7 +19,6 @@ namespace AzFramework
         {
             serializeContext->Class<PaintBrushSettings>()
                 ->Version(6)
```

Comment:
Should this removal prompt a version change?

Reply:
Probably, though it's semi-irrelevant. The version number is only used by slices, not prefabs, and removing fields generally "just works" anyways. I can bump it just for cleanliness either in this change though if I need to resubmit anything or in my next change.

Reply:
Bumped!